unittests/feed/FeedTestUtil.h

```diff
@@ -55,20 +59,39 @@ class FeedBaseTest : public SyncAsioContextTest, public MockBackendTest {
         MockBackendTest::TearDown();
         SyncAsioContextTest::TearDown();
     }
+};
+
+class SharedStringJsonEqMatcher {
```

Comment:
Custom matcher to verify the send function is called with the expected parameter.

Reply:
This is not very important in unit tests but this is a good candidate for `detail` namespace. You can then create it thru the non-detail `SharedStringJsonEq` function.